pokemon-app/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import '../../styles/header.scss'
+
+const Header: React.FC = () => (
+  <header className='f-c'>
+    <Link to='/'><div className='logo'>Pokemonix</div></Link>
+  </header>
+)
+
+export default Header
```

Comment:
New line expected

Reply:
You also can add a rule to your .eslintrc if you have it to automate adding new line.